app/internal/features.py

```diff
@@ -0,0 +1,169 @@
+from fastapi import Depends
+
+from app.features.index import features
+from app.database.models import UserFeature, Feature
+from app.internal.utils import create_model, get_current_user
+from app.dependencies import get_db, SessionLocal
+
+
+def create_features_at_startup(session: SessionLocal):
```

Comment:
Add type annotations

Reply:
To what?
The function only parameter is session and it has a type annotations.